generate_grpc.sh

```diff
@@ -11,15 +11,15 @@ if [ ! -e ./vendor_proto/protobuf/src/google/protobuf ]; then
 fi
 
 # controller
-protoc -I pkg/engine/controller/rpc/pb -I vendor_proto/protobuf/src/ pkg/engine/controller/rpc/pb/controller.proto --go_out=plugins=grpc:pkg/engine/controller/rpc/pb
+protoc -I pkg/engine/controller/rpc/pb -I vendor_proto/protobuf/src/ pkg/engine/controller/rpc/pb/controller.proto --go_out=plugins=grpc:./../../../
```

Comment:
This makes less sense than $GOPATH.

If $GOPATH doesn't exist, how can we know what's at `./../../../`?

Reply:
As long as the developers follow the convention of go code organization, `./../../../` should be the parent directory of `github.com/`.

Reply:
The convention you mentioned is GOPATH. User doesn't need to put directory there using go mod.